flake.nix

```diff
@@ -2,10 +2,7 @@
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
     flake-utils.url = "github:numtide/flake-utils";
-    crane = {
-      url = "github:ipetkov/crane";
-      inputs.nixpkgs.follows = "nixpkgs";
-    };
+    crane.url = "github:ipetkov/crane";
```

Comment:
LGTM. I was skeptical about removing `crane.inputs.nixpkgs.follows = "nixpkgs";`, but apparently they no longer rely on `nixpkgs` directly (that's what the warning/update was about, duh).

Reply:
Yes, they no longer pin the version; so following is no longer necessary